### ISSUE TITLE
Login Form not Available

### ISSUE THREAD

**Issue by tauqueerdanish** [adamspd/django-appointment#248]
## Feature Description
The login form is not available through which i can login as admin or end user. Due to this no proper front end is developing at my end. Help me in this regard.

**Comment by adamspd** [comment]
Hi @tauqueerdanish, the login form available is the one given by django, I didn't add one since every user has their own preferences.

**Comment by tauqueerdanish** [comment]
and site is also down where we can see the demo.

**Comment by adamspd** [comment]
Yeah, the demo site is down, I was supposed to fix a problem on it 5 days ago but didn't have enough time.
The thing is, the package is for use in other projects that already have a login page for user. Adding a login page would be overkill...
What I could do though, on the page where I say login to be able to go through the appointment process faster would be to put the default login page variable provided by django, so if the user sets it in settings, it would use that and if login is successful, would use the user's information to prefill the appointment.
But this would take time. If you know how to code, perhaps you could do a pull request ?

**Comment by tauqueerdanish** [comment]
ok thank you. any idea when the demo site is up?

**Comment by adamspd** [comment]
I'm working on it. I will try my best to make it work by tomorrow night.

**Comment by adamspd** [comment]
Hi @tauqueerdanish, the [demo site](https://django-appt.adamspierredavid.com/) is up and running.

**Comment by adamspd** [comment]
Hi @tauqueerdanish, I’m going to close this issue now. If you run into anything else, just open a new one. Thanks!
